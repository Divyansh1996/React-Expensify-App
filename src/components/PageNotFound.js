import React from "react";
import {Link} from 'react-router-dom';

const PageNotFound = () => (
    <p>This is a Not Found 404 Page - <Link to="/">Go to Home Page</Link></p>
)

export default PageNotFound;