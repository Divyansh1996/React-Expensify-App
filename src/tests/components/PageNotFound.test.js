import React from "react";
import {shallow} from "enzyme";
import PageNotFound from "../../components/PageNotFound"

test("Testing the PageNotFound page", () => {
    const wrapper = shallow(<PageNotFound/>)
    expect(wrapper).toMatchSnapshot()
})