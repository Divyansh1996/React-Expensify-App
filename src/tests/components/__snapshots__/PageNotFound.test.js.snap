// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Testing the PageNotFound page 1`] = `
<p>
  This is a Not Found 404 Page - 
  <Link
    to="/"
  >
    Go to Home Page
  </Link>
</p>
`;
